app/components/verkiezingen/draft-mandataris-list.hbs

```diff
@@ -0,0 +1,101 @@
+<div class="au-c-body-container au-c-body-container--scroll">
```

Comment:
I think it is a bit dangerous to put this in the component, I would put this in the template, then we have more control in the template if we want to put the scroll on the page or on the table.